### PR TITLE
add ip accounting metrics

### LINKED ISSUE #23: add ip accounting metrics
starting with v2.35 systemd supports the following ip accounting metrics available via the d-bus apis:
* ingress bytes
* egress bytes
* ingress packets
* egress packets

background:
http://0pointer.net/blog/ip-accounting-and-access-lists-with-systemd.html

I don't mind submitting a PR for this

### PR DESCRIPTION
**Pull request by jwhited**
closes #23

### CODE REVIEW

File: systemd/systemd.go [hunk 1]
```diff
@@ -164,6 +169,27 @@ func NewCollector(logger log.Logger) (*Collector, error) {
...
+		"Service unit ingress IP accounting in bytes.",
+		[]string{"name"}, nil,
+	)
+	ipEgressBytes := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_egress_bytes"),
+		"Service unit egress IP accounting in bytes.",
+		[]string{"name"}, nil,
+	)
+	ipIngressPackets := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_ingress_packets"),
```
Comment (povilasv) [inline systemd/systemd.go:184]:
```suggestion
		prometheus.BuildFQName(namespace, "", "service_ip_ingress_packets_total"),
```

as per https://prometheus.io/docs/practices/naming/

As per

File: systemd/systemd.go [hunk 2]
```diff
@@ -164,6 +169,27 @@ func NewCollector(logger log.Logger) (*Collector, error) {
...
+		"Service unit ingress IP accounting in packets.",
+		[]string{"name"}, nil,
+	)
+	ipEgressPackets := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_egress_packets"),
```
Comment (povilasv) [inline systemd/systemd.go:189]:
```suggestion
		prometheus.BuildFQName(namespace, "", "service_ip_egress_packets_total"),
```

**Comment by jwhited** [comment]
> Thanks. Just one nit, otherwise LGTM!

Thanks Povilas, hopefully this is good now. Thanks for creating this collector.

File: systemd/systemd.go [hunk 3]
```diff
@@ -164,6 +169,27 @@ func NewCollector(logger log.Logger) (*Collector, error) {
 		"Resident memory size in bytes.",
 		[]string{"name"}, nil,
 	)
+
+	ipIngressBytes := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_ingress_bytes_total"),
```
Comment (povilasv) [inline systemd/systemd.go:174]:
```suggestion
		prometheus.BuildFQName(namespace, "", "service_ip_ingress_bytes"),
```

We didn't need to add `_total` to `bytes` as bytes is a unit ;)

File: systemd/systemd.go [hunk 4]
```diff
@@ -164,6 +169,27 @@ func NewCollector(logger log.Logger) (*Collector, error) {
...
+		"Service unit ingress IP accounting in bytes.",
+		[]string{"name"}, nil,
+	)
+	ipEgressBytes := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_egress_bytes_total"),
```
Comment (povilasv) [inline systemd/systemd.go:179]:
```suggestion
		prometheus.BuildFQName(namespace, "", "service_ip_egress_bytes"),
```

**Pull request merged**